src/Discord.Net.Core/Entities/Messages/IUserMessage.cs

```diff
@@ -84,6 +84,23 @@ public interface IUserMessage : IMessage
         /// <seealso cref="IEmote"/>
         Task RemoveReactionAsync(IEmote emote, IUser user, RequestOptions options = null);
         /// <summary>
+        ///     Removes a reaction from message.
+        /// </summary>
+        /// <example>
+        ///     The following example removes the reaction, <c>💕</c>, added by the message author from the message.
+        ///     <code language="cs">
+        ///     await msg.RemoveReactionAsync(new Emoji("\U0001f495"), 84291986575613952);
+        ///     </code>
+        /// </example>
+        /// <param name="emote">The emoji used to react to this message.</param>
+        /// <param name="userId">The id of the user that added the emoji.</param>
```

Comment:
```suggestion
        /// <param name="userId">The ID of the user that added the emoji.</param>
```
keeping this consistent with other docs